### PR TITLE
#185467897 Tailwind configuration

### PR DESCRIPTION
**Pull request by frankflej**
## What does this PR do?
This PR sets necessary cofinguration to help you start using tailwind
## Description of Task to be completed?
This task focuses on setting up up tailwind which is an easy and reliable css framework used in which helps in shortening our css codebase and easy responsiveness
## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout ch-tailwind`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm run start`

- And try adding css in shortcut used in tailwind like the one used in `*/src/containers/AboutContainer.jsx`

## Screenshots (if appropriate)
Screen Shot 2015-09-21 at 8 35 04 PM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
-    "dev": "vite",
+    "start": "vite",
```
Comment (nishimweprince) [inline package.json:7]: Excellent!!

**Review by Dyves1 — APPROVED** [review]
Great job 👏

**Review by nishimweprince — APPROVED** [review]
Great work!!

**Review by bergad37 — APPROVED** [review]
Well done!

**Review by irakevine — APPROVED** [review]
Great work

**Review by jcharles921 — APPROVED** [review]
Great 👍

**Review by niyontwali — APPROVED** [review]

**Pull request merged**